src/main/java/frc/robot/commands/revolver/RunRevolver.java

```diff
@@ -28,6 +28,7 @@ public RunRevolver(double revolverSpeed, double wheelSpeed, Revolver revolver) {
   // Called when the command is initially scheduled.
   @Override
   public void initialize() {
+    
```

Comment:
Please remove this change.

Generally, please try to avoid any changes unrelated to your main goal. You can see all the changes in your PR by pressing the "Files Changed" tab at the top of your PR.

Reply:
@LukasC27 This change is still required